src/lm_polygraph/utils/manager.py

```diff
@@ -174,6 +176,14 @@ def estimate_uncertainty(
     return UncertaintyOutput(ue[0], input_text, texts[0], model.model_path)
 
 
+def _flatten_estimates(e, calculator_class):
```

Comment:
The naming here is a bit confusing: the function is called flatten_estimates, if accepts calculator class, and then it's used to flatten generation metrics.
Concepts of estimator, calculator and generation metric a fairly well defined in our code.

Since this is a universal function that can flatten whateverthefuck, suggest we rename this to something like `_flatten_results`, arguments to `(results, result_generator_class)`, and change naming in return statement to something along the same lines.

It would also benefit from the short docstring that explains that this can flatten any type of result (ues, stats and gen_metrics), and a comment before the return statement that expected shape of the `results` is `[num inputs, num_token_level_results_per_input]`